eosdis-nasa/earthdata-pub-api
# Add get step endpoints

# Description

Added get steps endpoints to provide consistent UI with `/questions` endpoint.

## Spec

See Ticket: [EDPUB-1386](https://bugs.earthdata.nasa.gov/browse/EDPUB-1386)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Use swagger to query GET for `/api/data/steps` endpoint and GET for `/api/data/steps/{id}` endpoint and validate that you receive the expected return.
